.github/workflows/rebase-default-branch.yml

```diff
@@ -0,0 +1,59 @@
+# Rebases a pull request on the repo's default branch when the "rebase" label is added
+# Link: https://github.com/Infomaniak/.github/blob/main/workflow-templates/rebase-default-branch.yml
+
+name: Rebase Pull Request
+
+on:
+  pull_request:
+    types: [ labeled ]
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+env:
+  DEFAULT_BRANCH: ${{ github.event.repository.default_branch }}
+
+jobs:
+  main:
+    if: ${{ contains(github.event.*.labels.*.name, 'rebase') }}
+    name: Rebase
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v5.0.0
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          fetch-depth: 0
+
+      # Link: https://httgp.com/signing-commits-in-github-actions/
+      - name: Import bot's GPG key for signing commits
+        id: import-gpg
+        uses: crazy-max/ghaction-import-gpg@v6.3.0
```

Comment:
For this third-party GitHub Action, can we put the version in a comment, and use a commit sha instead, to ensure reproducibility?